package/src/frontend/VideoPlayer/filterSources.js

```diff
@@ -1,31 +1,35 @@
-import $ from 'jquery';
 import {browser} from '../browser';
 
 export const filterSources = function(playerElement) {
-  if (!$(playerElement).is('video')) {
+  if (playerElement.tagName !== 'Video') {
     return playerElement;
   }
 
   var changed = false;
 
   if (browser.has('mp4 support only')) {
     // keep only mp4 source
-    $(playerElement).find('source').not('source[type="video/mp4"]').remove();
+    playerElement.querySelectorAll('source').forEach(source => {
```

Comment:
Just something to be aware of: `querySelectorAll` returns a `NodeList` - not an `Array` - which does not support `forEach` in older browsers. Apparently we already have a polyfill for this, through one of our dependencies.